students/16th/team1/yeonu_hong/user/views.py

```diff
@@ -0,0 +1,121 @@
+import json
+import bcrypt
+import jwt
+from django.http  import JsonResponse
+from django.views import View
+from my_settings  import SECRET
+from decorator    import login_check
+from .models      import User, Follow
+
+# 회원가입
+class SignUpView(View):
+    def post(self, request): 
+        try:
+            data  = json.loads(request.body)
+            # 셋 중 하나만 들어와도 가입이 진행되어야하는데 무조건 keyError로 넘어가버림
+            # 들어오는 값만 변수에 지정하기 어떻게 하지?!
+            name  = data['name']
+            phone = data['phone']
+            email = data['email']
+            password         = data['password'].encode('utf-8')
+            password_encoded = bcrypt.hashpw(password, bcrypt.gensalt())
+            password_decoded = password_encoded.decode('utf-8')
```

Comment:
너무 깔끔하게 구현해주셨네요 👍
하지만 이런 부분은 이제 프로세스를 아시기 때문에 아래처럼 한줄로 줄여서 써주셔도 좋을 것 같아서 남겨드립니다.
```suggestion
            hashed_password = bcrypt.hashpw(password, bcrypt.gensalt()).decode('utf-8')
```

Reply:
넵!!